config/locales/en.yml

```diff
@@ -1684,5 +1684,8 @@ en:
       validation:
         invalid_user_assigned_to_work_package:
           "The chosen user is not allowed to be '%{property}' for this work package."
+        estimated_hours:
+          "Estimated hours cannot be set on parent work packages."
```

Comment:
The format "already present" is a single line, that would otherwise be longer than 100 characters btw...